docs/ui/schema-conv/sharded-migration.md

```diff
@@ -35,3 +35,15 @@ The `migration_shard_id` column is populated via the database to shardId map tha
 
 ![](https://services.google.com/fh/files/misc/shard_schema.png)
 
+## `migration_shard_id` as Primary Key
+It is a common practice to use auto-incremented primary keys in many source databases. However during a sharded migration, where
+multiple source shards are merged into a single Spanner Database, this may lead to conflicts during the data migration. To avoid
+these conflicts, the `migration_shard_id` is added to the primary key of each table by default. 
```

Comment:
Should we add details like "This column is auto-populated using the logical_shard_id provided in input" ?

Reply:
This is mentioned in the previous point